superset-frontend/src/dashboard/components/SliceHeaderControls.jsx

```diff
@@ -87,6 +87,8 @@ const RefreshTooltip = styled.div`
   color: ${({ theme }) => theme.colors.grayscale.base};
 `;
 
+const screenshotNodeSelector = '.dashboard-component-chart-holder';
```

Comment:
super nit: since this is a constant, prefer `SCREENSHOT_NODE_SELECTOR`

Reply:
Done